xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -246,6 +246,14 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
 ##
 ##
 ##
+#macro(displayCommentButton)
+<button type="button" id="OpenCommentForm" class="btn btn-default">
+  $services.icon.renderHTML('comment')
+    $services.localization.render('core.viewers.comments.add.comment.label')</button>
```

Comment:
I see some indentation issues here. Is it possible to handle any layout need in CSS?

Reply:
Indentation fixed